src/index.css

```diff
@@ -2,7 +2,6 @@
 @import url('https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200');
 @import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
 
```

Comment:
- [ ] I will implore you to style the app exactly as the [Zeplin](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378/stage) app is. As of now, the background color is different.

### Zeplin design
![image](https://user-images.githubusercontent.com/86200356/222431365-7574225b-e35e-4337-a2b6-c06d132e1f69.png)
### Your design
![image](https://user-images.githubusercontent.com/86200356/222431770-cd0ad39b-7ca7-445b-bc3b-5891d197857a.png)